src/xrGame/GamePersistent.cpp

```diff
@@ -38,6 +38,7 @@
 #include "xrCore/Threading/TaskManager.hpp"
 
 #include "xrPhysics/IPHWorld.h"
+#include "DiscordRPC.h"
```

Comment:
Please, remove `AnselManager.h` inclusion on line 37 and make it like that:
```suggestion
#include "xrPhysics/IPHWorld.h"

#include "AnselManager.h"
#include "DiscordRPC.h"
```